<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Remastersys</class>
 <widget class="QMainWindow" name="Remastersys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>715</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>715</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Remastersys</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>691</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="remastersys.qrc">:/images/remastersys-large.png</pixmap>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>661</width>
      <height>321</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QConsoleWidget" name="console1" native="true">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>16</y>
        <width>641</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="mainImg">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>221</width>
        <height>221</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="remastersys.qrc">:/images/main.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QConsoleWidget" name="console2" native="true">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>16</y>
        <width>641</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>10</y>
        <width>181</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="splashImgBtn">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Splash Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="grubImgBtn">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Grub Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="plymouthBtn">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Plymouth Theme</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="userSettingBtn">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>User Settings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="configureBtn">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Configure</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>202</x>
        <y>10</y>
        <width>451</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Select a 640x480 png file for the live menu background</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Select a picture for the Grub menu background </string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Create and select a plymouth theme</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Copy user settings to /etc/skel and be the default for all new users</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Customize the settings for remastersys</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>221</width>
        <height>221</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="remastersys.qrc">:/images/main.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>16</y>
        <width>631</width>
        <height>301</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>150</y>
      <width>641</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="backupBtn">
       <property name="text">
        <string>Backup</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="distBtn">
       <property name="text">
        <string>Distribution</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="customBtn">
       <property name="text">
        <string>Customize</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearBtn">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="checklogBtn">
       <property name="text">
        <string>Check Log</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="quitBtn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>540</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>540</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QConsoleWidget</class>
   <extends>QWidget</extends>
   <header>terminalwidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="remastersys.qrc"/>
 </resources>
 <connections/>
</ui>
